.env.example

```diff
@@ -1,2 +1,3 @@
 NEXT_PUBLIC_OAK_ASSETS_HOST=
 NEXT_PUBLIC_OAK_ASSETS_PATH=
+NEXT_PUBLIC_CLOUDINARY_CLOUD_NAME=
```

Comment:
Annoyingly this is needed by `next-cloudinary`. It's the first part of `NEXT_PUBLIC_OAK_ASSETS_PATH` so perhaps there is an opportunity to remove `NEXT_PUBLIC_OAK_ASSETS_PATH` since the `/image/upload` portion appears to be constant to all Cloudinary accounts?

I guess this will be a breaking change